Makefile

```diff
@@ -6,50 +6,37 @@ help:
 	@echo
 	@echo "Available targets are:"
 	@echo " help                    show this text"
-	@echo " build			builds basic development environment"
+	@echo " build                   builds basic development environment"
 	@echo " clean                   remove python bytecode and temp files"
 	@echo " clean-dev               also cleans up development environment"
-	@echo " cli-test		run cli sanity tests"
-	@echo " full-dev		builds full development environment"
+	@echo " deploy                  builds and starts basic development environment"
+	@echo " full-build              builds full development environment"
+	@echo " full-deploy             builds and starts basic development environment"
```

Comment:
This seems to be a copy/paste error: `basic` vs. `full`

Should we rather say what the difference between them is so that the help text provides any value?

Reply:
Nice catch!  Addressed in  ddbd30ba5e5bc68e0e258a95e57f5df4dc10eb12.